### DISCUSSION TITLE
timestamp column should be first in schema auto-created from Influx Line Protocol

### DISCUSSION THREAD

**Discussion by comunidadio** [questdb/questdb#1272] [Ideas]
**Describe the bug**
It is rather uncommon and inconvenient to have timestamp as last column in auto-created schemas.
For tables with many columns, the timestamp not even visible from console without scrolling.

**To Reproduce**
1. Write a new measurement through ILP port 9009 on a table that does not exist (yet)
2. Notice timestamp is last column in autocreated schema

**Expected behavior**
Timestamp is the first column of autocreated schema.

**Comment by bluestreak01** [comment]
You might want to rally the troops for this one. I feel this is subjective.

**Comment by bluestreak01** [comment]
We might close this unless this thread turns into a discussion. There are workarounds, table can be created upfront with the desired order of columns

**Comment by sunng87** [comment]
We might want to add document for table schema auto-generation of influxdb line-protocol. This feature is missing from [the docs](https://questdb.io/docs/develop/insert-data#influxdb-line-protocol). It's a key feature to consider before we actually using the line protocol.

**Reply by bluestreak01** [reply]
good point, we shall do that
